### ISSUE TITLE
Force _updated and _created values

### ISSUE THREAD

**Issue by tomaluca95** [pyeve/eve#1108]
Hi,

Is possible to create/update an object forcing _updated and _created values?

I'm running a migration from a legacy system and would be nice to have al data with the right timestamp.

I tried to add `updated` or `_updated` to the payload but i get 422 Client Error

**Comment by nicolaiarocci** [comment]
Nope. What you could do, however, is update those fields offline directly on mongo once the migration is complete.

**Comment by tomaluca95** [comment]
Thanks,

how I can re-calculate the etag to match the new data?

**Comment by nicolaiarocci** [comment]
The quickest way out would be to also drop the etag field after the migration: it will be re-computed when it is missing.

**Comment by tomaluca95** [comment]
Thanks :+1:
